src/main/java/seedu/address/ui/ScheduleCard.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.person.Person;
+
+/**
+ * An UI component that displays information of a {@code Schedule}.
+ */
+public class ScheduleCard extends UiPart<Region> {
+
+    private static final String FXML = "ScheduleListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Person person;
+
+    @FXML
+    private HBox scheduleCardPane;
+    @FXML
+    private Label name;
+    @FXML
+    private Label id;
+    @FXML
+    private Label phone;
+    @FXML
+    private Label address;
+
```

Comment:
Empty line here.

Reply:
Thanks for pointing it out! Will resolve